.version

```diff
@@ -1 +1 @@
-0.4.0
\ No newline at end of file
+0.5.0
```

Comment:
Do we need a major version jump here, as I don't *think* we are changing the public API?

Reply:
Well, technically this is a minor version jump according to semver.